### PR TITLE
Migrating docstrings from cpython to pythonland

### PR DESCRIPTION
**Pull request by bocajnotnef**
as per #1185

Join me on my journey to liberate the docstrings from the oppression of the cpython! Together we can return them to their glorious homeland!

### CODE REVIEW

File: khmer/__init__.py [hunk 1]
```diff
@@ -287,4 +287,27 @@ class HLLCounter(_HLLCounter):
     """
 
     def __len__(self):
-        return self.estimate_cardinality()
+        """Return the current estimation."""
+        return self._estimate_cardinality()
+
```
Comment (mr-c) [inline khmer/__init__.py]: Please don't rename the CPython methods. Just call the method directly `return self.estimate_cardinality()`

**Comment by standage** [comment]
Has recent reorganization rendered this PR unnecessary?

**Comment by betatim** [comment]
Unsure, but I would wait for the cythonisation and then add docstrings to that instead of moving from cpython to python by refreshing this PR.

**Comment by standage** [comment]
Closing as no longer relevant.

**Pull request closed without merging**